docs/src/pages/components/data-grid/style/StripedGrid.js

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { alpha, styled } from '@mui/material/styles';
+import { DataGrid, GridRow, gridClasses } from '@mui/x-data-grid';
+import { useDemoData } from '@mui/x-data-grid-generator';
+import clsx from 'clsx';
+
+const ODD_OPACITY = 0.2;
+
+const StripedDataGrid = styled(DataGrid)(({ theme }) => ({
+  [`& .${gridClasses.row}.even`]: {
+    backgroundColor: '#EEEEEE',
+    '&:hover, &.Mui-hovered': {
+      backgroundColor: alpha(theme.palette.primary.main, ODD_OPACITY),
```

Comment:
```suggestion
      backgroundColor: '#E0E0E0',
```
The hover should move from light to darker gray. Here I propose the hover moves from grey-200 to grey-300 according to https://material.io/resources/color/#!/?view.left=0&view.right=0&primary.color=bcbcbc

Reply:
How do you make the hover and / or selected consistent with this color ?